test/runner/lib/classification.py

```diff
@@ -348,13 +348,19 @@ def _classify(self, path):
                 'network-integration': target.name if 'network/' in target.aliases else None,
             }
 
-        if path.startswith('test/integration/'):
+        if path.startswith('test/integration/network-'):
             return {
-                'integration': 'all',
-                'windows-integration': 'all',
                 'network-integration': 'all',
             }
 
+        if path.startswith('test/integration/windows-'):
+            return {
+                'windows-integration': 'all',
+            }
+
+        if os.path.dirname(path) == 'test/integration' and path.endswith('.template'):
```

Comment:
This is incorrect, as changes to `.template` files affect integration tests. Since changes to templates are infrequent, I wouldn't worry about adding special handing here either.